nautobot/extras/forms/forms.py

```diff
@@ -544,6 +546,20 @@ def perform_pre_delete(self, queryset):
 #
 # Custom Links
 #
+class CustomLinkBulkEditForm(BootstrapMixin, NoteModelBulkEditFormMixin):
```

Comment:
Should `content_type` be bulk editable as well?

Reply:
No, the `add_`/`remove_` pattern is only applicable for many-to-many. Here it can just be a single form field.